sdk/appcenter/src/test/java/com/microsoft/appcenter/persistence/DatabasePersistenceTest.java

```diff
@@ -191,8 +220,8 @@ public void getLogsWithGetCorruptedIdsException() throws Exception {
         /* Mock instances. */
         mockStatic(AppCenterLog.class);
         DatabaseManager databaseManager = mock(DatabaseManager.class);
-        whenNew(DatabaseManager.class).withAnyArguments().thenReturn(databaseManager);
         when(databaseManager.nextValues(any(Cursor.class))).thenCallRealMethod();
+        DatabasePersistence persistence = createDatabasePersistenceInstance(mock(Context.class), null, null, databaseManager, null);
```

Comment:
There is a log of `mock(Context.class)`, let's add class field with annotation for it